akamas/1.1.0/README.md

```diff
@@ -0,0 +1,136 @@
+# Akamas Webhook Integration
+
+This integration enables developers, performance engineers, DevOps, and SREs to get their applications automatically optimized in terms of performance, resilience, and cost efficiency (according to any defined application-specific goal), when these applications are released into staging or other testing environments. The integration of Akamas with Keptn ensures that services provide the highest quality at the lowest possible cost.
+
+## Prerequisites 
+
+1. Install Akamas (make sure you have a valid license).
+2. Port 6000 must be open on the Akamas server. 
+3. Install the Akamas pre-built container. This container will act as a bi-directional bridge between Akamas Studies and Keptn webhooks. 
+
+    <details>
+    <summary>How to install the Akamas container</summary>
+
+    - Make an application directory such as /usr/local/keptn
+    - In the application directory, download the files below.
+      ```bash
+      curl https://akamas.s3.us-east-2.amazonaws.com/integrations/keptn/docker-compose.yml -o docker-compose.yml
```

Comment:
The container name `keptn-webhook` is not very specific. Maybe change it to something like `keptn-akamas-webhook-service`.

Reply:
done